cypress/e2e/main.cy.ts

```diff
@@ -0,0 +1,13 @@
+describe("Homepage tests", () => {
+  it("Console is cleared of errors and warnings", () => {
+    cy.visit("/", {
+      onBeforeLoad(win) {
+        cy.stub(win.console, "warn").as("consoleWarn");
+        cy.stub(win.console, "error").as("consoleError");
+      },
+    });
+    cy.get("@consoleWarn").should("not.be.called");
```

Comment:
Warn is probably okay but console errors and thrown errors probably not.

Reply:
Gotcha will just check for errors then.